pkg/manager/member/pd_member_manager.go

```diff
@@ -393,17 +393,21 @@ func (pmm *pdMemberManager) syncTidbClusterStatus(tc *v1alpha1.TidbCluster, set
 			Health:    memberHealth.Health,
 		}
 
-		oldPDMember, exist := tc.Status.PD.PeerMembers[name]
-
-		status.LastTransitionTime = metav1.Now()
-		if exist && status.Health == oldPDMember.Health {
-			status.LastTransitionTime = oldPDMember.LastTransitionTime
-		}
+		var exist bool
+		var oldPDMember v1alpha1.PDMember
 		if pattern.Match([]byte(clientURL)) {
+			oldPDMember, exist = tc.Status.PD.Members[name]
 			pdStatus[name] = status
 		} else {
+			oldPDMember, exist = tc.Status.PD.PeerMembers[name]
 			peerPDStatus[name] = status
 		}
+
+		status.LastTransitionTime = metav1.Now()
+		if exist && status.Health == oldPDMember.Health {
+			status.LastTransitionTime = oldPDMember.LastTransitionTime
```

Comment:
Can the change here update the values in pdStatus or peerPDStatus?

Reply:
can`t!  fixed